2-notify-owner-on-ticket-to-prod-assist/ticket-status-prod-assist/code/src/functions/ticket_stage_change/utils/devrev-utils.ts

```diff
@@ -32,13 +36,15 @@ export async function getPartOwnersString(partObject: any) {
     return partOwnersString;
 }
 
-export async function ticketTimelineEntryCreate(ticketID: string, body: string, token: string) {
-    const timelineEntryCreatePath = "timeline-entries.create";
-    let endpoint = DEVREV_API_BASE + timelineEntryCreatePath;
+export async function ticketTimelineEntryCreate(ticketID: string, body: string, token: string, api_base: string ) {
+    const devrevSDK = client.setup({
+        endpoint: api_base,
+        token: token,
+    })
     let payload: any = {
         object: ticketID,
         type: "timeline_comment",
         body: body,
     }
-    await postCall(endpoint, payload, token);
-}
+    await devrevSDK.timelineEntriesCreate(payload);
+}
```

Comment:
what is the benefit of this over direct SDK usage?

Reply:
What does direct SDK usage mean?
Do you mean, directly using the SDK in place of the function - `ticketTimelineEntryCreate` ? This is just done for the separation of concern.